lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -181,3 +181,12 @@ function gutenberg_enqueue_global_styles() {
 add_action( 'wp_footer', 'gutenberg_enqueue_global_styles', 1 );
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_stored_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_stored_styles', 1 );
+
+function gutenberg_enqueue_classic_theme_styles() {
+	if ( ! wp_is_block_theme() ) {
+		wp_register_style( 'classic-theme-styles', gutenberg_url( 'build/block-library/classic.css' ), array(), true );
+		wp_enqueue_style( 'classic-theme-styles' );
+	}
+}
+add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_classic_theme_styles' );
+add_action( 'admin_enqueue_scripts', 'gutenberg_enqueue_classic_theme_styles' );
```

Comment:
As this is unusual, I would favour adding a comment here to explain _why_ we needed the additional `admin_enqueue_scripts`.

Reply:
Done!

Reply:
Sorry my comment wasn't enormously helpful. I could have made a suggestion in Github and done some work for you.

Thanks for following up 👍